src/twisted/newsfragments/10310.misc

```diff
@@ -0,0 +1 @@
+When using a `Deferred` with a canceller that references the `Deferred` (ex. `DelayedCall` and `inlineCallbacks`), a canceller reference from the `Derefferd` is removed a soon as the `Deferred` is called. In this way, the canceller should have a higher change of being garbage collected.
```

Comment:
@erikjohnston 

Is this a .misc, like ... don't bother letting other people know about this change.

`.bugfix` -- like things were broken in previous Twisted version , see if you might be affected and consider an update

`.feature` -- only consider an update if you plan make to use of this improvement

Reply:
I think the only people that care about this are applications with large and non standard memory usage. You could argue for any of the three tbh, but I think few people will care?

For Synapse I think we'd consider this a misc change (but we still include those in the changelogs)

Reply:
Twisted doesn't include the text of misc fragments in the news file.